### DISCUSSION TITLE
Add responsive value to prop size

### DISCUSSION THREAD

**Discussion by VHQDevFS** [csandman/chakra-react-select#241] [Ideas]
Can support prop size like this?

`
<ChakraReactSelect size={{base: 'sm', 'md':'md'}} />
`

**Comment by csandman** [comment]
This is a pretty good idea! It definitely makes sense to replicate the same behavior that Chakra uses for their `Input` component.

I actually already pushed a new version with this feature, in [`v4.5.0`](https://github.com/csandman/chakra-react-select/releases/tag/v4.5.0).  You can check out a demo here: https://codesandbox.io/s/xtxbq1?file=/app.tsx

Let me know if you have any issues with it!

**Reply by VHQDevFS** [reply]
Thank a lot! I love this package.

**Reply by csandman** [reply]
I'm glad you like it! I've put a ton of time into it at this point haha
